packages/trimerge-sync/src/TrimergeClient.ts

```diff
@@ -116,6 +136,7 @@ export class TrimergeClient<
       computeRef,
       getLocalStore,
       addNewCommitMetadata,
+      docCache,
```

Comment:
does it auto-infer types?
```suggestion
      docCache = new InMemoryDocCache(),
```